zenodo/modules/records/data/include/v4.1/datacite-relationType-v4.1.xsd

```diff
@@ -16,6 +16,8 @@
       <xs:enumeration value="IsSupplementedBy"/>
       <xs:enumeration value="IsContinuedBy"/>
       <xs:enumeration value="Continues"/>
+      <xs:enumeration value="IsDescribedBy"/>
```

Comment:
This file we don't need to modify... We used to validate at some point our DataCite export, but actually don't do that anymore.

I'll create an issue from here to actually remove these files completely.

Reply:
Ok! Wil discard theses changes then 👍